installation/resources/installer-config-local.yaml.tpl

```diff
@@ -211,3 +211,16 @@ data:
   autoscaler_hpa.resources.requests.memory: "100Mi"
   controller.resources.requests.cpu: "30m"
   controller.resources.requests.memory: "100Mi"
+apiVersion: v1
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: core-tests
```

Comment:
it's not `core` anymore, and all other configmaps have the `-overrides` suffix, so I'd suggest to rename it to `console-overrides`